plugins/jsonp-highlight/prism-jsonp-highlight.js

```diff
@@ -39,15 +39,16 @@
 	 * @returns {Adapter} A registered adapter or `null`.
 	 */
 	function getAdapter(adapter) {
+		var i = 0, item;
```

Comment:
You don't need to initialize `i`. The for loops do that already.

```suggestion
		var i, item;
```